# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/moz.build
# Commit: 86c478d114ec
# Full Hash: 86c478d114ec223ecf3698ddbb52977bf786f10c
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2020-05-11 21:47:06
# Regressor Bug: 1618287
# File Overlap Count: 1
# Description:
#   Bug 1618287 - PreloaderBase class, r=smaug
#   
#   Depends on D71800
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67479
# ==============================================================================

diff -r 5c64c1dfb318 -r 86c478d114ec uriloader/moz.build
--- a/uriloader/moz.build	Mon May 11 14:03:19 2020 +0000
+++ b/uriloader/moz.build	Mon May 11 14:20:39 2020 +0000
@@ -13,4 +13,5 @@
     'base',
     'exthandler',
     'prefetch',
+    'preload',
 ]